mergekit/plot_tools/plot_tools.py

```diff
@@ -0,0 +1,410 @@
+import numpy as np
+from typing import List, Dict, Optional, Any, Tuple
+from mergekit.graph import Task
+import networkx as nx
+import plotly.graph_objects as go
```

Comment:
We should capture these new dependencies in `pyproject.toml`. Probably under a feature, so headless installs don't need to bring them in.